packages/federatorai/overlay/questions.yaml

```diff
@@ -0,0 +1,80 @@
+rancher_min_version: 2.2.0
```

Comment:
Please remove this, this constraint isn't supported in this repository.

Reply:
Fixed by forcibly commit.